code/game/objects/structures/fusioncell.dm

```diff
@@ -0,0 +1,88 @@
+/obj/structure/fusioncell
+	name = "Fusion Cell"
+	desc = "A simple way to store radioactive energy with many capabilities."
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "oldshieldoff"
+	density = FALSE
+	anchored = FALSE
+	max_integrity = INFINITY
+	var/emitting = FALSE
+	var/radiation_count = 0
+	var/emittingmount = 0
+	var/i
+	var/datum/looping_sound/geiger/soundloop
+
+
+	/obj/structure/fusioncell/attackby(obj/item/I, mob/user, params)
+		if(I.tool_behaviour == TOOL_WRENCH)
+			default_unfasten_wrench(user, I, time = 5)
+		return
+
+	/obj/structure/fusioncell/proc/update_sound()
```

Comment:
Reason why linter fails is that you have for some procs a single space infront of obj remove the space for all the effected procs and linter should do just fine lines effected are line 16, line 21, line 28, line 63, line 76, line 81, line 87 remove the space infront of every single one and linter should stop screaming at you